h/templates/app.html

```diff
@@ -16,6 +16,7 @@
           --><span class="provider" ng-show="auth.user">/{% raw %}{{ auth.user|persona:'provider' }}{% endraw %}</span><!--
           --><i class="h-icon-triangle"></i></span>
         <ul class="dropdown-menu pull-right" role="menu">
+          <li share-this-page="" ng-hide="isStream"></li>
```

Comment:
I cannot get this to work for some reason. Any ideas? Is it on the $parent scope for some reason? Do I need to modify my directive / controller to have access to `isStream` or `isEmbedded`?